CMakeLists.txt

```diff
@@ -154,9 +164,17 @@ if (BUILD_SHARED_LIBS)
                                              SOVERSION ${OPENTRACING_VERSION_MAJOR})
   target_compile_definitions(opentracing PRIVATE OPENTRACING_EXPORTS)
   install(TARGETS opentracing EXPORT OpenTracingTargets
-          RUNTIME DESTINATION ${LIB_INSTALL_DIR}
-          LIBRARY DESTINATION ${LIB_INSTALL_DIR}
-          ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
+          RUNTIME DESTINATION ${LIB_INSTALL_DIR}64
+          LIBRARY DESTINATION ${LIB_INSTALL_DIR}64
+          ARCHIVE DESTINATION ${LIB_INSTALL_DIR}64
```

Comment:
The idea with LIB_INSTALL_DIR is that it's set from the command line to override the default location when you invoke cmake, e.g.,
```
cmake -DLIB_INSTALL_DIR=lib64 ..
```
So could you remove the `64` suffixes and use `-DLIB_INSTALL_DIR=lib64` if you want to build the package.